Дз4/css/style_for_float.css

```diff
@@ -0,0 +1,170 @@
+div {
+    text-align: center;
+}
+
+.header1 {
+    background-color: #DCDCDC;
+    float: left;
+    width: 33%;
+    height: 50px;
+}
+
+.header2 {
+    background-color: #D3D3D3;
+    float: left;
+    width: 33%;
+    height: 50px;
+}
+
+.header3 {
+    background-color: #C0C0C0;
+    float: left;
+    width: 34%;
+    height: 50px;
+}
+
+.clearfix:after {
+    content: " ";
+    clear: both;
```

Comment:
this is not working
![image](https://user-images.githubusercontent.com/47972692/169030257-7e0a736e-cdfd-4829-b706-ea84b00c3868.png)